### DISCUSSION TITLE
Support for late interaction and sparse models

### DISCUSSION THREAD

**Discussion by S1LV3RJ1NX** [StarlightSearch/EmbedAnything#89] [Q&A]
Do we support late interaction models like ColBERT? Or Sparse models like BM25?

**Comment by sonam-pankaj95** [comment]
We support Sparse Models, BM25 is a search method. In late interaction we support ColPali

**Comment by sonam-pankaj95** [comment]
@S1LV3RJ1NX In the recent release, we also support ColBERT and ModernBERT on ONNX .
